cna/docusaurus.config.js

```diff
@@ -151,8 +151,12 @@ const config = {
                 to: '/blog',
               },
               {
-                label: 'Contribute to this page',
-                href: 'https://github.com/cloud-native-austria/cna-website'
+                label: 'Impressum',
```

Comment:
```suggestion
                label: 'Imprint',
```

Reply:
@thschue should there really be an inconsistency between the naming of label of the link in the footer and the actual title on the page?